src/main/java/io/fabric8/maven/docker/config/handler/property/PropertyConfigHandler.java

```diff
@@ -79,7 +79,8 @@ private boolean buildConfigured(BuildImageConfiguration config, ValueProvider va
         return valueProvider.getString(FROM, config == null ? null : config.getFrom()) != null ||
                 valueProvider.getMap(FROM_EXT, config == null ? null : config.getFromExt()) != null ||
                 valueProvider.getString(DOCKER_FILE, config == null || config.getDockerFileRaw() == null ? null : config.getDockerFileRaw()) != null ||
-                valueProvider.getString(DOCKER_FILE_DIR, config == null || config.getDockerArchiveRaw() == null ? null : config.getDockerArchiveRaw()) != null;
+                valueProvider.getString(DOCKER_FILE_DIR, config == null || config.getDockerArchiveRaw() == null ? null : config.getDockerArchiveRaw()) != null ||
+                valueProvider.getString(DOCKER_FILE_DIR, config == null || config.getDockerFileDirRaw() == null ? null : config.getDockerFileDirRaw()) != null ;
```

Comment:
Good catch. Actually this method should also check for a `Dockerfile` in the project base directory as this will also trigger a build without any configuration (the so called _simple Dockerfile mode_). 

Could you add this, too ? 

e.g.

```java
boolean buildConfigured = valueProvider...
if (buildConfigured) {
    return true;
}
return new File(project.getBasedir(),"Dockerfile").exists();
```

Reply:
PR updated